### ISSUE TITLE
H-P bonds (not in parm99/parm@Frosst)

### ISSUE THREAD

**Issue by bannanc** [open-forcefield-group/smirnoff99Frosst#21]
Looking at GAFF2 for inspiration. It has a dedicated hydrogen type for hydrogen bound to phosphorous (hp). 

Here are the H-P bonds:
```
hp-p2  266.54   1.336                     SOURCE3      87    0.1706
hp-p3  200.57   1.412             SOURCE3_SOURCE5     123    0.0510
hp-p4  253.60   1.349                     SOURCE3      17    0.1577
hp-p5  196.39   1.418             SOURCE3_SOURCE5      31    0.0077
```

Relevant phosphorous atom types
```
p2 30.97         1.538               Phosphate with two connected atoms
p3 30.97         1.538               Phosphate with three connected atoms, such as PH3
p4 30.97         1.538               Phosphate with three connected atoms, such as O=P(CH3)2
p5 30.97         1.538               Phosphate with four connected atoms, such as O=P(OH)3
```

I'm having a hard time making a decision about the hydrogen-phosphorous bond, but it seems like it would be better to keep just 1 H-P bond for now.

**Comment by davidlmobley** [comment]
In the absence of any deep chemical insight on my part, I'm tempted to go with a length of about 1.38 and a force constant which is the average of the ones listed. :) But if you 

This is one it would be interesting to revisit when we explore partial bond order -- I wonder if the force constant and length differences GAFF2 is attempting to capture here are something which is captured by the semiempirical calculation so we could just interpolate smoothly...

**Comment by bannanc** [comment]
@davidlmobley I pretended like I remembered o-chem yesterday and tried to draw out some structures and think about what would make the H-P bond stronger. Naively I assume something in the p2 and p4 types makes it more likely to polarize the bond making it stronger, otherwise H and P have similar electronegativities. I also tried to make guesses about which of these phosphorous is more common. It seems like there are possibly two distinct groups here, I think it makes more sense to round one of them knowing that might be bad for some case and good for others, instead of being bad for everything. 

```
[#1:1]-[#15:2]    200.    1.4
```
